### ISSUE TITLE
support hold play/pause?

### ISSUE THREAD

**Issue by bazzargh** [postlund/pyatv#1178]
**What feature would you like?**

An action equivalent to hold play/pause while on home screen. allows selecting output device https://www.macrumors.com/guide/top-10-apple-tv-remote-tips-tricks/ (item 10)

**Describe the solution you'd like**

Possibly play/pause needs to support input actions as was added for other buttons in https://github.com/postlund/pyatv/commit/25d1f4b732087278bcd2a0ccef09c764cfba06d7 ? I don't know what the controller is sending over the air in this case

**Any other information to share?**

I came here looking to see if this feature was supported as oddly it doesn't work in the iphone controller application, only with the physical controller.

**Comment by postlund** [comment]
Interesting, didn't know about this. I can add it to the play button as well, if you want to try it for me? If it works, then I'll make sure to merge it.

**Comment by bazzargh** [comment]
sure. I wasn't sure where to add it in the code for myself, it seemed like play/pause were handled differently from the other buttons you'd enabled that for. If you figure something out and have a patch/pr/whatever I can test it

**Comment by postlund** [comment]
Yeah, it's a bit special as it tries to handle the case where PlayPause isn't natively supported by the active player. If you want, you could just try hardcoding the hold action for play and pause and see what happens. Probably the quickest way to get feedback.

**Comment by postlund** [comment]
I'm working on this, will add support for it soon unless I encounter any issues.

**Comment by postlund** [comment]
It's however possible to change output devices programmatically via the audio interface now (should add some documentation about that). So this might not be needed to the same extent (as I assume that was the use case)?

**Comment by postlund** [comment]
So, I just tried this on my Apple TV and it seems that this shortcut does not work anymore. Neither my iPhone nor the Apple Remote seems to work. Can you verify this too, @bazzargh?
